tools/oss-fuzz.sh

```diff
@@ -33,7 +33,7 @@ if [ -z "$FUZZING_ENGINE" ]; then
 fi
 
 meson $build -D$fuzzflag -Db_lundef=false
-ninja -C $build fuzzers
+ninja -v -C $build fuzzers
```

Comment:
This will generate a lot of noise. Is this needed?

Reply:
The fuzzers are kind of fragile in the sense that they heavily rely on the right options passed to `clang`. Without looking at how they are built it's hard to tell whether they're in good shape (plus some environments where the script is run (like Azure Pipeline) aren't interactive so the output of ninja without `-v` looks really ugly (and is difficult to decipher) there. In general I think `-v` most certainly should be used in "CI" mode. (and I'm not sure at this point it would make sense to complicate the script to make it less verbose everywhere else)